netboot-services/ipxeMenuGenerator/ipxe-menu-generator.env

```diff
@@ -1 +1,7 @@
 NETBOOT_SERVER_IP="IP of the server the TFTP server is running on"
+AZURE_NETBOOT_SERVER_IP="IP of the Azure Netboot Server"
+AZURE_SYNC_SAS_TOKEN="SAS Token for fallback on Storage Account"
```

Comment:
I thought, that the reverse proxy will be our fallback?

Reply:
That wouldn't give us a big benefit as of now. We don't want to migrate the storage account to switzerland and the connection to azure from several locations is slow. First we want to capitalize on our own netbootserver and if it's unreachable, we go back to azure.